tests/test_experiment.py

```diff
@@ -60,6 +63,20 @@ def test_model_prefix(test_dir: str) -> None:
     assert model._key_prefixing_enabled == True
 
 
+def test_model_no_name():
+    exp = Experiment("test_model_no_name")
+    with pytest.raises(AttributeError):
+        _ = exp.create_model(name=None, run_settings=RunSettings("python"))
+
+
+def test_model_no_name():
```

Comment:
This test needs to be renamed to `test_ensemble_no_name`; it's currently overriding the test above it.

Reply:
good catch! Thank you